lib/OpenQA/Script/CloneJob.pm

```diff
@@ -193,8 +193,10 @@ sub create_url_handler ($options) {
         api => $local_url->host,
         apikey => $options->{apikey},
         apisecret => $options->{apisecret});
-    die "API key/secret for '$options->{host}' missing. Checkout '$0 --help' for the config file syntax/lookup.\n"
-      unless $local->apikey && $local->apisecret;
+    if (!$options->{'export-command'}) {
+        die "API key/secret for '$options->{host}' missing. Checkout '$0 --help' for the config file syntax/lookup.\n"
+          unless $local->apikey && $local->apisecret;
+    }
```

Comment:
```suggestion
    die "API key/secret for '$options->{host}' missing. Checkout '$0 --help' for the config file syntax/lookup.\n"
          unless $options->{'export-command'} || ($local->apikey && $local->apisecret);
```

Reply:
I personally never use `unless` with multiple conditions because it makes my brain sick.